doc/minimal-manual.nix

```diff
@@ -0,0 +1,95 @@
+# Minimal manual prototype
+
+{ pkgs ? (import ../. {}), nixpkgs ? { }} :
+
+with pkgs;
+
+let
+
+  rootPath = "http://localhost:8000/result";
+
+  rewriteHeader = ''s|# ([^{]*)\s\{(#([^}]*))}$|# <span id="\3">[\1](\2)</span>|g'';
+  rewriteAdmonitionStart = ''s|::: warning|<div class="warning"><h3 class="title">Warning</h3><p>\n|g'';
```

Comment:
I think we use note admonition as well at least, here is a full list:

https://github.com/jgm/pandoc/pull/6922/files#diff-d3c4d5818e408935dae861295b6e9b73434ea63069e7b08d2c06ece876f271a7R198

Reply:
added support for all the admonition types.